sql/hive/src/main/scala/org/apache/spark/sql/hive/hiveWriterContainers.scala

```diff
@@ -70,8 +70,11 @@ private[hive] class SparkHiveWriterContainer(
   @transient protected lazy val committer = conf.value.getOutputCommitter
   @transient protected lazy val jobContext = newJobContext(conf.value, jID.value)
   @transient private lazy val taskContext = newTaskAttemptContext(conf.value, taID.value)
-  @transient private lazy val outputFormat =
-    conf.value.getOutputFormat.asInstanceOf[HiveOutputFormat[AnyRef, Writable]]
+  @transient private lazy val outputFormat = conf.value.getOutputFormat match {
```

Comment:
outputFormat is only used in `[Utilities.getFileExtension(conf.value, fileSinkConf.getCompressed, outputFormat)](https://github.com/weiqingy/spark/blob/0fa2bb791d1fa9c37fe89c1942ce0ed950a9ee59/sql/hive/src/main/scala/org/apache/spark/sql/hive/hiveWriterContainers.scala#L96)` to get the file extension. If `outputFormat` is "null", the extension will be "".